### ISSUE TITLE
[BUG] Minor coral bug(s)

### ISSUE THREAD

**Issue by emeraldleo1982** [ParadoxGamingRP/bugs-and-suggestions#2050]
### Your Discord Name

jojosnmustard

### A clear and concise description of what the bug is

When you get to your diving location, you're told there's 20 jobs to do instead of 10, which leads to the marker not going away once you've completed the 10.

### Steps to Reproduce the Bug

Get coral job
Get told your workload is doubled
Workload is not doubled

### Expected Behavior

Get told how many coral you actually need to harvest.

### Screenshots/Videos

![image](https://github.com/ParadoxGamingRP/bugs-and-suggestions/assets/29337315/5d3bd84b-807e-4b6d-bcc1-7febdfb29716)

Also seeing this, not sure how it's related, but it's showing up while coraling, not when talking with the guy or interacting with the tool.

### Extra Notes (Optional)

The marker not going away **doesn't** seem to be affecting future location assignments even when I don't "Stop this job" before getting a new one.

**Comment by Darahz** [comment]
Yeah i did a stupid thing... Sorry about that, fixed next restart

**Comment by emeraldleo1982** [comment]
Thank you, we appreciate all your hard work :)
